### ISSUE TITLE
mjpegReqHandler.write

### ISSUE THREAD

**Issue by leoneckert** [gluxon/node-mjpeg-server#5]
it seems like the `mjpegReqHandler.write` method only works when I clone the module straight from github rather than npm install it...is that correct?
sorry just curious, if the case i don't include this in my package.json but make a note to install manually

**Comment by gluxon** [comment]
Node.js requires should work regardless of where the source was downloaded from. I know the example in the READEME has the below snippet since I copied it from the examples folder,

```js
var mjpegServer = require('..');
```

but this should be used instead in your projects.

```js
var mjpegServer = require('mjpeg-server');
```

**Comment by leoneckert** [comment]
oh yes the `require('mjpeg-server')` bit works without problem. I am just building something with this and some other modules. All of the others I just `npm install`. But for yours `npm install` seems to give me a slightly different version than when I clone it straight from this repo. 

Not a big problem, just a heads up that the two versions seem to be different. Great module though, thanks!!

**Comment by gluxon** [comment]
Oh, you're right. Somehow npm still has `v0.2.1` instead of `v0.3`. I just published v0.3 which uses streams2 now. Thanks for the bug report. Let me know if there's still issues after that.
